setup.py

```diff
@@ -49,16 +49,16 @@ def get_all_extras() -> Dict:
 all_extras = get_all_extras()
 
 base_deps = [
-    "base58>=1.0.3",
-    "jsonschema>=3.0.0",
-    "packaging>=20.3",
-    "semver>=2.9.1",
+    "base58>=1.0.3,<3.0.0",
```

Comment:
why `3.0.0` and not `2.0.0`?

Also, since the current version Is `2.1.0` and we are sure it works, maybe we could add the lower bound?